views/package.json

```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.8.1",
-        "@oat-sa/tao-item-runner-qti": "1.6.5"
+        "@oat-sa/tao-item-runner-qti": "https://github.com/oat-sa/tao-item-runner-qti-fe.git#integration/AUT-22/inline_choice_interaction_allow_non-text_options"
```

Comment:
This is intentional? I didn't know that was possible if it works. Just asking because it looks strange to change version number with url.